### PR TITLE
Some improvements to the documentation I've been meaning to make

### PR DESCRIPTION
**Pull request by Avaq**
See the commit message bodies for change descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,25 +35,50 @@ For more information:
...
-  try the [bundled module](#global-bundle-cdn) to avoid a package manager.
-- For older browsers, you can use a bundler such as [Rollup][] or WebPack.
-  Fluture doesn't use ES5+ language features, so the source does not have to
-  be transpiled. Alternatively, you can use the
-  [CommonJS Module](#commonjs-module).
+  try the [bundled module](#bundled-from-a-cdn) to avoid a package manager.
+- For older browsers, use a bundler such as [Rollup][] or WebPack. Fluture
+  doesn't use ES5+ language features, so the source does not have to be
+  transpiled. Alternatively, there is a [CommonJS Module](#commonjs-module)
+  available.
```
Comment (davidchambers) [inline README.md:81]: What do you mean by ES5+ language features, exactly? Fluture may not use ES5 language features, but I would be surprised to discover that it doesn't use any of the functions and methods specified in ES5 ([`Object.keys`][1], for example).


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys

Comment (Avaq) [reply README.md:81]: > Fluture may not use ES5 language features

I'm talking about *post* ES5 language features, as (perhaps badly) indicated by the plus symbol.

> What do you mean by ES5+ language features, exactly?

I mean language features specifically, as opposed to newly added API's. So syntax.
As for newly added APIs, there's a comment about them further down in the README:

https://github.com/fluture-js/Fluture/pull/431/files#diff-04c6e90faac2675aa89e2176d2eec7d8R102-R103

Comment (davidchambers) [reply README.md:81]: To me, _ES5+_ means `>= ES5` rather than `> ES5`. Perhaps you could describe Fluture as _ES5-compatible_ to avoid confusion.

Comment (Avaq) [reply README.md]: How is this? https://github.com/fluture-js/Fluture/pull/431/commits/2e719949311f88667921dac5a4550f8947a507b6

Comment (davidchambers) [reply README.md]: That commit looks good. :)

**Pull request merged**